batch_seqr_loader/batch_workflow.py

```diff
@@ -181,16 +210,19 @@ def main(
         + '.dict',
     )
 
-    ped_check_j = _pedigree_checks(
-        b=b,
-        samples_df=samples_df,
-        reference=reference,
-        sites=b.read_input(SOMALIER_SITES),
-        ped_file=b.read_input(ped_fpath),
-        work_bucket=work_bucket,
-        overwrite=overwrite,
-        fingerprints_bucket=join(data_bucket, 'fingerprints'),
-    )
+    ped_check_j = None
+    if not skip_ped_checks:
+        ped_check_j, ped_fpath = _pedigree_checks(
+            b=b,
+            samples_df=samples_df,
+            reference=reference,
+            sites=b.read_input(SOMALIER_SITES),
+            ped_file=b.read_input(ped_fpath),
+            overwrite=overwrite,
+            fingerprints_bucket=fingerprints_bucket,
+            web_bucket=web_bucket,
+            web_url=f'https://{namespace}-web.populationgenomics.org.au/{project}',
+        )
 
     # realign_bam_jobs = _make_realign_bam_jobs(
```

Comment:
Accidental leftover?

Reply:
Keeping it as a placeholder, will be implemented in a following PR.